<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>747</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupIdeBox">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="ideBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="ideBoxPage1">
           <attribute name="icon">
            <iconset>
             <normaloff>../../Game-build-desktop/pixmaps/models.png</normaloff>../../Game-build-desktop/pixmaps/models.png</iconset>
           </attribute>
           <attribute name="title">
            <string/>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="groups"/>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="buttonsBox">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>115</width>
                 <height>346</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_8" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="buildButtonsLayout">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QCheckBox" name="useSnap">
                   <property name="text">
                    <string>Сетка</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="pause">
                   <property name="iconSize">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>игрок</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="curPl">
                   <item>
                    <property name="text">
                     <string>Нейтрал</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>3</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>4</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>5</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>6</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>7</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>8</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="icon">
            <iconset>
             <normaloff>../../Game-build-desktop/pixmaps/mountains.png</normaloff>../../Game-build-desktop/pixmaps/mountains.png</iconset>
           </attribute>
           <attribute name="title">
            <string/>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="2">
               <widget class="QToolButton" name="groundDown">
                <property name="icon">
                 <iconset>
                  <normaloff>../../Game-build-desktop/pixmaps/mountains_down.png</normaloff>../../Game-build-desktop/pixmaps/mountains_down.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QToolButton" name="groundNo">
                <property name="icon">
                 <iconset>
                  <normaloff>../../Game-build-desktop/pixmaps/ecursor.png</normaloff>../../Game-build-desktop/pixmaps/ecursor.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="groundUp">
                <property name="icon">
                 <iconset>
                  <normaloff>../../Game-build-desktop/pixmaps/mountains_up.png</normaloff>../../Game-build-desktop/pixmaps/mountains_up.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QSlider" name="brushSize">
                 <property name="maximum">
                  <number>30</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QToolButton" name="setGrass">
                    <property name="text">
                     <string>g</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="setSand">
                    <property name="text">
                     <string>s</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string/>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <widget class="QLabel" name="fps">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="dipCount">
                <property name="text">
                 <string>dipCount</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="deltaTime">
                <property name="text">
                 <string>dt</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="bathCount">
                <property name="text">
                 <string>bathCount</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>411</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="mainSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>500</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="layMiniMap"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="topMargin">
        <number>7</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="stackedWidget_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="stackedWidget_2Page1">
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="unitsLay"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page">
               <layout class="QHBoxLayout" name="horizontalLayout_5"/>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>200</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="unitAvatarLayout"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>215</width>
           <height>200</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="layControl"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>координаты</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_5" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QDoubleSpinBox" name="posX">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="posY">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="posZ">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>размеры</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_7" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QDoubleSpinBox" name="sizeX">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                   <property name="value">
                    <double>1.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="sizeY">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                   <property name="value">
                    <double>1.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="sizeZ">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                   <property name="value">
                    <double>1.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_6" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>поворот</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rotOZ">
                   <property name="maximum">
                    <double>360.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>10.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
